lib/definitions/retry.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Default exponential backoff configuration for retries.
+ */
+const RETRY_CONF = {
+  retries: 3,
+  retryAfterBaseValue: 1000,
+  doNotRetry: [400, 401, 403],
```

Comment:
After switching to your branch (thanks for that) we still see rate-limits, but they always seem to return with a 403.

Any reason why 403s where excluded?

<img width="640" alt="image" src="https://user-images.githubusercontent.com/864578/227858643-b42a5d3a-ed92-462d-8ea7-2c41e5fcc21b.png">

Reply:
Yes, maybe that's the issue I ran into. I noticed that rate limiting wasn't retrying.

Reply:
Nor sure if you still use the instruction provided in https://github.com/semantic-release/github/pull/487#issuecomment-1377986211, but for us it started to work after actually defining an override in our `package.json`:

```
"overrides": {
    "semantic-release": {
      "@semantic-release/github": "$@semantic-release/github"
    }
}
```

Reply:
I believe the secondary rate limit error is caused by this: https://github.com/octokit/plugin-throttling.js/issues/454

This may be a short-term solution - https://github.com/semantic-release/github/pull/487#issuecomment-1486298997 but I think it'll turn off the (rather nice) feature that adds a comment to each PR in a given release saying "this PR is included in release version XXX"